Blockzilla/IntroViewController.swift

```diff
@@ -26,6 +26,104 @@ struct IntroViewControllerUX {
     static let FadeDuration = 0.25
 }
 
+protocol PageControlDelegate {
+    func selectedIndex(_ index:Int)
+}
+
+class PageControl: NSObject {
+    
+    var currentPage = 0
+    var numberOfPages = 0
+    var hidesForSinglePage = false
+    
+    var stack : UIStackView? = nil
```

Comment:
should be a `let` I think. nit: space before colon. 
Optional: in UI code, feel free to do UIStackView! (implicit unwraps), because Apple started that convention. (BTW, implicit unwraps are avoided outside of UI code).